BrainPortal/app/views/tools/_tools_table.html.erb

```diff
@@ -112,12 +112,12 @@
     end
 
     t.column("Help", :help) do |tool|
-      path = tool.cbrain_task_class.constantize.public_path("edit_params_help.html") rescue nil
+      path = tool.cbrain_task_class.public_path("edit_params_help.html") rescue nil
```

Comment:
Nope, it's correct.

In the old code, there was a method `cbrain_task_class()` which returned a string. It needed to be `constantize()d` to create the class on which to invoke `public_path()`.

In the new code, `cbrain_task_class()` returns the class itself. I could have changed the code to

```
path = tool.cbrain_task_class_name.constantize.public_path("edit_params_help.html") rescue nil
```

but that would have been ugly, especially since like I said the new `cbrain_task_class()` already does that.